homeassistant/components/nuki/lock.py

```diff
@@ -26,29 +27,32 @@
 NUKI_DATA = 'nuki'
 
 SERVICE_LOCK_N_GO = 'nuki_lock_n_go'
-SERVICE_UNLATCH = 'nuki_unlatch'
+SERVICE_CHECK_CONNECTION = "nuki_check_connection"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
-    vol.Required(CONF_TOKEN): cv.string
+    vol.Required(CONF_TOKEN): cv.string,
+    vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int
 })
 
 LOCK_N_GO_SERVICE_SCHEMA = vol.Schema({
     vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
     vol.Optional(ATTR_UNLATCH, default=False): cv.boolean
 })
 
-UNLATCH_SERVICE_SCHEMA = vol.Schema({
+CHECK_CONNECTION_SERVICE_SCHEMA = vol.Schema({
     vol.Optional(ATTR_ENTITY_ID): cv.entity_ids
 })
 
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Nuki lock platform."""
     from pynuki import NukiBridge
-    bridge = NukiBridge(config.get(CONF_HOST), config.get(CONF_TOKEN))
-    add_entities([NukiLock(lock) for lock in bridge.locks])
+    bridge = NukiBridge(config.get(CONF_HOST), config.get(CONF_TOKEN),
```

Comment:
Please use `dict[key]` for required config keys and keys with default config schema values.

Reply:
I'm sorry, but I don't know what you mean... Where can I find an example?

Reply:
`config[CONF_HOST]` instead of `config.get(CONF_HOST)`. We only need to use `dict.get` if we don't know if the key is in the dict or not.

Reply:
Done